locator/tablets.cc

```diff
@@ -134,6 +134,21 @@ tablet_migration_streaming_info get_migration_streaming_info(const locator::topo
                 return result; // No nodes to stream to -> no nodes to stream from
             }
 
+            if (features.repair_based_tablet_rebuild) {
```

Comment:
The decisions should be based only on group0 state, not features. If feature is enabled during transition, topology coordinator algorithms will be confused, decisions at different stages will be inconsistent.

Features can be used to choose which transition to start.